jsr352/core/src/main/java/org/hibernate/search/jsr352/massindexing/impl/util/MassIndexingPartitionProperties.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.jsr352.massindexing.impl.util;
+
+
+/**
+ * @author Yoann Rodiere
+ */
+public final class MassIndexingPartitionProperties {
```

Comment:
Let's keep `MassIndexingPartitionProperties`, which would be consistent with `MassIndexingParameters` in term of naming style.